docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -259,7 +259,7 @@ Network object specifies service IPAM as *external*:
     spec:
       type: flat
```

Comment:
please help to change type to "vpc" in this sample; "flat" has very little sense to have external ipam - this was defect in the original doc.

Reply:
will update.